packages/story-editor/src/app/story/utils/isEmptyStory.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { shallowEqual } from '@googleforcreators/react';
+
+/**
+ * Internal dependencies
+ */
+import { DEFAULT_PAGE_BACKGROUND_COLOR } from '../../../elements/utils/createPage';
+
+const isEmptyStory = (pages) => {
```

Comment:
Worth turning this into a `useIsEmptyStory()` hook?

Reply:
For me - not -> https://github.com/GoogleForCreators/web-stories-wp/pull/10261#discussion_r788212048, but maybe I don't see some of the pros?